### PR TITLE
Fix #238

### LINKED ISSUE #238: [1.7.10]Crash when searching item in Custom Stuff 3
Here is the crash report.
https://paste.ee/p/rUyeR

### PR DESCRIPTION
**Pull request by SeraphJACK**
Formated

### CODE REVIEW

File: src/main/java/binnie/genetics/item/ItemSerum.java [hunk 1]
```diff
@@ -93,7 +95,10 @@ public GeneItem getGeneItem(ItemStack stack) {
 	public String getItemStackDisplayName(ItemStack stack) {
 		IGeneItem gene = getGeneItem(stack);
 		BreedingSystem breedingSystem = Binnie.Genetics.getSystem(gene.getSpeciesRoot());
-		return I18N.localise("genetics.item.serum.name", breedingSystem.getDescriptor());
+		if (breedingSystem != null) {
+			return I18N.localise("genetics.item.serum.name", breedingSystem.getDescriptor());
+		}
+		return I18N.localise("genetics.item.serum.name");
```
Comment (KorDum) [inline src/main/java/binnie/genetics/item/ItemSerum.java]: It works without formatted args?

`genetics.item.serum.name=%s Serum`

**Review by KorDum — CHANGES_REQUESTED** [review]

**Comment by SeraphJACK** [comment]
Changed

File: src/main/resources/assets/genetics/lang/en_US.lang [hunk 2]
```diff
@@ -27,7 +27,9 @@ genetics.item.database.1.name=Master Gene Database
 genetics.item.registry.0.name=Registry
 genetics.item.registry.1.name=Master Registry
 genetics.item.serum.name=%s Serum
+genetics.item.serum.empty=Empty Serum
```
Comment (KorDum) [inline src/main/resources/assets/genetics/lang/en_US.lang]:
```
genetics.item.serumEmpty.name=Empty Serum Vial
genetics.item.genomeEmpty.name=Empty Serum Array
```
Please use existing keys

**Review by KorDum — CHANGES_REQUESTED** [review]

**Comment by SeraphJACK** [comment]
Changed

**Comment by SeraphJACK** [comment]
Changed

**Review by KorDum — APPROVED** [review]

**Comment by KorDum** [comment]
Thanks for fix!

**Pull request merged**